scripts/test-check-packages.sh

```diff
@@ -90,7 +90,7 @@ fi
 
 if [ "${PACKAGE_TEST_TYPE:-other}" == "with-kind" ]; then
   # Boot up the kind cluster
-  kind create cluster --config "$PWD/scripts/kind-config.yaml"
+  kind create cluster --config "$PWD/scripts/kind-config.yaml" --image "kindest/node:${K8S_VERSION}"
```

Comment:
Update the command to be the same as in the integrations repository. This will ensure that the k8s node versions are the same as the one set in the environment variable.